### PR TITLE
SR2021: Issue 03

### LINKED ISSUE #8: [SRAWN-SUBMIT] SR2021 Issue 3
```
sryear: SR2021
issue: 03
submission-date: 2021-02-27T00:00:00Z
publication-date: 2021-02-28T12:00:00Z
```
## SR(A)WN Submission Thread

Student Robotics (Almost) Weekly Newsletter (aka. SR(A)WN) is a regular newsletter published by Student Robotics volunteers for volunteers.

- Keep up with what's going on in SR by subscribing to the SR(A)WN mailing list.
- Let others know about what we've been up to by submitting content to the newsletter.

### Content Submission

Anybody within SR can submit content to the newsletter by commenting on this thread.

The deadline for this issue is midnight UTC on 2021-02-27. Any content submitted after this deadline will be included in the following issue instead.

Please read the submission guidelines before submitting content to SR(A)WN.

<details>
<summary>Submission Guidelines</summary>

- Be inclusive! This is a newsletter for all volunteers, so please try not to assume knowledge (technical, or of specific activities within SR).
- Please follow the [code of conduct](https://opsmanual.studentrobotics.org/about-the-charity/code-of-conduct) when writing and submitting content.
- SR(A)WN is a public newsletter. Please don't submit any sensitive or inappropriate content.
- If additional help is required for something, you can **embolden** it to indicate this to volunteers.
- Please use [Commonmark Markdown](https://commonmark.org/help/) where possible when writing content. Some formatting may be stripped or changed during editing, or by the newsletter template.
- Please use alt text when submitting images to the newsletter, as some mail clients may not be able to read HTML.
</details>

### PR DESCRIPTION
**Pull request by trickeydan**
Closes #8

There are no notable updates from the Kit Team for this edition, so this section has been excluded.

### CODE REVIEW

File: SR2021/2020-02-28-srawn-03.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Headlines
+
+### Mentoring
+
+**There are a number of teams who need mentors to help them with this year's game.** No experience with mentoring or the simulator required, but experience with SR is definitely useful!
+
+If you're interested, please let the competition team know!
+
+### Module III League
```
Comment (PeterJCLaw) [inline SR2021/2020-02-28-srawn-03.md:11]: Would be super awesome to use the Roman numeral here (and elsewhere) when referring to the modules:
```suggestion
### Module Ⅲ League
```
The league sessions however are numbered using Arabic numerals.

File: SR2021/2020-02-28-srawn-03.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+We announced Module III at the league session, and teams have 6 weeks to react until the first league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```
Comment (PeterJCLaw) [inline SR2021/2020-02-28-srawn-03.md:13]: Maybe remove "first"? It's a little confusing as this won't be the first league session -- it's going to be the third (overall).
```suggestion
We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```

File: SR2021/2020-02-28-srawn-03.md (hunk 1 above)
Comment (trickeydan) [reply SR2021/2020-02-28-srawn-03.md:11]: TIL that these are included in unicode!

File: SR2021/2020-02-28-srawn-03.md (hunk 2 above)
Comment (trickeydan) [reply SR2021/2020-02-28-srawn-03.md:13]: Yeah, absolutely agree. Looks like this content was copy-pasted from a previous issue when it was submitted.

File: SR2021/2020-02-28-srawn-03.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+League session 3 is in just few weeks on 20th March, where teams will compete in the game defined by Module Ⅲ. Here we'll also unveil Module Ⅳ, the final module of SR2021, and what will be played in league session 4 and the knockouts.
+
+### Competition Doing: 6th March
+
+We'll be holding "Doing" on Saturday 6th of March from 2PM UK time to 6PM (but may last longer), working on various competition topics, such as updates to the competition software, wrapping up any changes to Module Ⅳ, and other smaller simulator fixes.
+
+## Competition
+
+### Module IV Design
```
Comment (PeterJCLaw) [inline SR2021/2020-02-28-srawn-03.md:23]:
```suggestion
### Module Ⅳ Design
```

File: SR2021/2020-02-28-srawn-03.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### Mentoring
+
+**There are a number of teams who need mentors to help them with this year's game.** No experience with mentoring or the simulator required, but experience with SR is definitely useful!
+
+If you're interested, please let the competition team know!
+
+### Module Ⅲ League
+
+We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```
Comment (PeterJCLaw) [inline SR2021/2020-02-28-srawn-03.md:13]:
```suggestion
We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](https://studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```
(slightly surprised that this link was working without this)

**Pull request merged**